tests/test/org/commcare/cases/util/test/CasePurgeFilterTests.java

```diff
@@ -60,73 +70,9 @@ public void setUp() throws Exception {
         e.setUserId(groupOwner);
     }
 
-    @Test
-    public void testOwnerPurge() {
-        try {
-            storage.write(a);
-            storage.write(b);
-            storage.write(c);
-            storage.write(d);
-
-            int[] present = new int[] {a.getID(), c.getID(), b.getID(), d.getID()};
-            int[] toRemove = new int[] {  };
-
-            Vector<Integer> removed = storage.removeAll(new CasePurgeFilter(storage, userOwned));
-            testOutcome(storage, present, toRemove);
-            testRemovedClaim(removed, toRemove);
-
-        } catch(Exception e) {
-            e.printStackTrace();
-            fail("Unexpected exception " + e.getMessage());
-        }
-    }
-
-    @Test
-    public void testUnownedPurge() {
-        b.setUserId(otherOwner);
-        try {
-            storage.write(a);
-            storage.write(b);
-            storage.write(c);
-            storage.write(d);
-
-            int[] present = new int[] {a.getID(), c.getID(), d.getID()};
-            int[] toRemove = new int[] { b.getID()};
-
-            Vector<Integer> removed = storage.removeAll(new CasePurgeFilter(storage, userOwned));
-            testOutcome(storage, present, toRemove);
-            testRemovedClaim(removed, toRemove);
-
-        } catch(Exception e) {
-            e.printStackTrace();
-            fail("Unexpected exception " + e.getMessage());
-        }
-    }
-
-    @Test
-    public void testOwnerLiveness() {
-        b.setUserId(otherOwner);
-        c.setUserId(otherOwner);
-
-        d.setIndex("b","b",b.getCaseId());
-
-        try {
-            storage.write(a);
-            storage.write(b);
-            storage.write(c);
-            storage.write(d);
-
-            int[] present = new int[] {a.getID(), b.getID(), d.getID()};
-            int[] toRemove = new int[] { c.getID()};
+    @RunWithResource("/case_relationship_tests.json")
+    public void runTests() {
```

Comment:
This method should be renamed to delineate the relationship with `CasePurgeTest`